README.md

```diff
@@ -0,0 +1,26 @@
+# dRep
```

Comment:
Add some information about dRep, or at least a link to the original repo, and maybe an attribution to the original creator(s) here. Something brief is fine